pinot-plugins/pinot-batch-ingestion/v0_deprecated/pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/HadoopSegmentPreprocessingJob.java

```diff
@@ -199,11 +291,31 @@ public void run()
       throw new RuntimeException("Job failed : " + job);
     }
 
-    _logger.info("Finished pre-processing job in {}ms", (System.currentTimeMillis() - startTime));
+    LOGGER.info("Finished pre-processing job in {}ms", (System.currentTimeMillis() - startTime));
+  }
+
+  private void fetchPreProcessingOperations() {
+    _preprocessingOperations = new HashSet<>();
+    TableCustomConfig customConfig = _tableConfig.getCustomConfig();
```

Comment:
We should not be parsing customConfigs in open source code. They are for the site to use as they see fit.

Reply:
> We should not be parsing customConfigs in open source code. They are for the site to use as they see fit.

Looks like we are already using it in the hadoop module. So, go ahead.